doc/releases/changelog-0.21.0.md

```diff
@@ -280,16 +273,18 @@
   The adjoint method requires memory for 4 independent state vectors, which corresponds roughly
   to storing a state vector of a system with 2 additional qubits.
 
-* A new method `qml.gradients.param_shift_hessian` has been added to directly compute the Hessian
-  (2nd order partial derivative matrix) of QNodes and QuantumTapes. The method generates
+<h4>New parameter-shift Hessian function</h4>
+
+* A new function `qml.gradients.param_shift_hessian` has been added to directly compute the Hessian
+  (2nd order partial derivative matrix) of QNodes and QuantumTapes. The function generates
```

Comment:
```suggestion
  (2nd order partial derivative matrix) of QNodes on hardware. The function generates
```

Reply:
let's also move the PR link here (after the full stop), and have a line break before the next paragraph :)

Reply:
Done thanks!